esp/src/src-react/hooks/deepHooks.ts

```diff
@@ -1,13 +1,13 @@
 import * as React from "react";
-import { deepEquals } from "@hpcc-js/util";
+import { verboseDeepEquals } from "@hpcc-js/util";
 
 //  Inspired by:  https://github.com/kentcdodds/use-deep-compare-effect
 
 function useDeepCompareMemoize<T>(value: T) {
     const ref = React.useRef<T>(value);
     const signalRef = React.useRef<number>(0);
 
-    if (!deepEquals(value, ref.current)) {
+    if (!verboseDeepEquals(value, ref.current)) {
```

Comment:
This was the change about the deepEquals. After the fact, I wasn't sure if I should've made a second function that used the verbose version and left this one alone?

Reply:
You don't want the "verbose" one, its only for debugging (maybe I should rename it)